.pipelines/ci.yml

```diff
@@ -25,12 +25,12 @@ variables:
     value: arosvcdev.azurecr.io/vpn
   - name: LOCAL_E2E_IMAGE
     value: arosvcdev.azurecr.io/e2e
-  - name: TAG
-    value: $(Build.BuildId)
   - name: VERSION
-    value: $(Build.BuildId)
+    value: $(TAG)
+  - name: TAG
+    value: pr-$(System.PullRequest.PullRequestId)-$(System.PullRequest.SourceCommitId)
```

Comment:
This pipeline also runs against the master branch directly, do we want to have a different tag for those builds? I expect that `System.PullRequest.PullRequestId` will not be set during those builds and will either cause issues or undesired names (e.g. `pr-null-asdhfjkl` or equivalent)